spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -116,7 +116,7 @@
 
     tags_link = find(".sidebar-section-link-sidebar-tags")
     categories_link = find(".sidebar-section-link-sidebar-categories")
-    tags_link.drag_to(categories_link)
+    tags_link.drag_to(categories_link, html5: true, delay: 0.4)
```

Comment:
I found out why delay was not working before. Only when html5 parameter is used, delay is taken into consideration (original `perform_legacy_drag` ignores delay)

https://github.com/teamcapybara/capybara/blob/348484b130e912a8657113a805162f0fa9e622ca/lib/capybara/selenium/extensions/html5_drag.rb#L14

Problem with html5 drag is that it is using modern html5 drag events instead of mouse events:

https://github.com/teamcapybara/capybara/blob/348484b130e912a8657113a805162f0fa9e622ca/lib/capybara/selenium/extensions/html5_drag.rb#L175

Therefore, new event listeners were added to draggable. That should be direction for the future anyway as drag events are better than mouse events.